ui/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessingTypeDeploymentService.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.engine.api.deployment
+
+import pl.touk.nussknacker.engine.api.ProcessVersion
+import pl.touk.nussknacker.engine.graph.EspProcess
+
+import scala.concurrent.Future
+
+trait ProcessingTypeDeploymentService {
+
+  // shouldn't we pass also category as a filter?
```

Comment:
I don't think so ;)

Reply:
I believe that u understand this association correctly - will remove that comment